### PR TITLE
Removed Open field from PollData

### LINKED ISSUE #252: Remove the Open field from the PollData
Currently the poll object contains a field names `Open` of type boolean that tells whether the poll is still open or not. Since there is already the `Ends` field telling when the post will end, I think that it's best to remove the `Open` field at all. This would make the whole code easier to handle. We would not need a `BeginBlocker` iterating over all the polls and setting that field automatically. Also, clients would rely only on one field to make sure if the poll is closed or not (the ending date). 

Inside the hanlder, we could simply verify that users are not voting to closed polls by using the `ctx.BlockTime` variable and comparing it with the poll ending date.

### PR DESCRIPTION
**Pull request by bragaz**
## Description
This PR remove the `Open` field from the `PollData` object.
Closes #252.

## Checklist
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Wrote unit tests.
- [ ] Wrote integration tests (simulation & CLI). 
- [x] Updated the documentation. 
- [x] Added an entry to the `CHANGELOG.md` file.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Comment by bragaz** [comment]
@RiccardoM I've solved conflict with master with this PR that was already been approved.
I think we can merge this after the checks

**Comment by bragaz** [comment]
The failed coverage test is due to the `computeID` thing.
I've also added another test case to raise the coverage a little bit.
@RiccardoM.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# Version 0.11.0
+## Changes
+- Changed postIDs computation. Now they are set inside the `NewPost` method instead of being required from it (#253)
+- Removed `Open` field from `PollData` to make the code easier to handle (#252)
```
Comment (RiccardoM) [inline CHANGELOG.md:4]:
```suggestion
- Removed the `Open` field from within the `PollData` object. Now you should rely on the `CloseDate` field to determine whether a poll is closed or open. (#252)
```

**Review by RiccardoM — APPROVED** [review]
LGTM :rocket:

**Pull request merged**